[34mThu May 28 00:13:03 2015 +0200[m [33m4c448a1[32m (HEAD, master)[m [34mfix [35m [Govinda Fichtner][m
diff --git a/scripts/build.sh b/scripts/build.sh
index 5f08688..768e38a 100755
--- a/scripts/build.sh
+++ b/scripts/build.sh
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 #
 # This script builds the application from source.
-set -e
+set -e -x
 
 # Get the parent directory of where this script is.
 SOURCE="${BASH_SOURCE[0]}"
@@ -40,12 +40,20 @@ fi
 # Install dependencies
 echo "--> Installing dependencies to speed up builds..."
 go get \
+  -a \
   -ldflags "${CGO_LDFLAGS}" \
+  -x \
+  -tags netgo \
+  -installsuffix nocgo \
   ./...
 
 # Build!
 echo "--> Building..."
 go build \
+    -a \
+    -x \
+    -tags netgo \
+    -installsuffix nocgo \
     -ldflags "${CGO_LDFLAGS} -X main.GitCommit ${GIT_COMMIT}${GIT_DIRTY} -X main.GitDescribe ${GIT_DESCRIBE}" \
     -v \
     -o bin/consul${EXTENSION}
